Update azure-kubernetes-service-extension.md (#2930)
dapr/docs: daprdocs/content/en/developing-applications/integrations/Azure/azure-kubernetes-service-extension.md

dapr-sidecar-injector-9555889bc-rqjgt   1/1     Running   0          1h
```

> For further information such as configuration options and targeting specific versions of Dapr, see the official [AKS Dapr Extension Docs](https://docs.microsoft.com/azure/aks/dapr).

Reviewer: ```suggestion
For more information about configuration options and targeting specific Dapr versions, see the official [AKS Dapr Extension Docs](https://docs.microsoft.com/azure/aks/dapr).
```
